server/webapp/WEB-INF/rails.new/app/assets/stylesheets/pipeline_configs/_pipeline.scss

```diff
@@ -343,3 +345,45 @@
     }
   }
 }
+
+
+//tasks
+.lookup{
```

Comment:
Please - no top level css. We're already in a lot of css mess because of things like this.